ts/components/conversation/message/message-item/MessageRequestResponse.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { PropsForMessageRequestResponse } from '../../../../models/messageType';
+import { getConversationController } from '../../../../session/conversations';
+import { UserUtils } from '../../../../session/utils';
+import { Flex } from '../../../basic/Flex';
+import { SpacerSM, Text } from '../../../basic/Text';
+import { ReadableMessage } from './ReadableMessage';
+
+export const MessageRequestResponse = (props: PropsForMessageRequestResponse) => {
+  const { messageId, isUnread, receivedAt, conversationId, source } = props;
+
+  let profileName = '';
+  if (conversationId) {
+    profileName =
+      getConversationController()
+        .get(conversationId)
+        .getProfileName() + '';
+  }
+  const msgText =
+    profileName && props.source === UserUtils.getOurPubKeyStrFromCache()
+      ? window.i18n('messageRequestAcceptedOurs', [profileName])
+      : window.i18n('messageRequestAccepted');
```

Comment:
can you use one of the selector or hook to fetch this profileName instead? It's just that set like that, this won't be updated if we set the nickname or if the profileName of that user changes before a refresh of the conversation.

Something like useConversationUsername() (search for it to see how it is used)